packages/manager/cypress/support/intercepts/object-storage.ts

```diff
@@ -478,3 +479,12 @@ export const interceptUpdateBucketAccess = (
     apiMatcher(`object-storage/buckets/${cluster}/${label}/access`)
   );
 };
+
+/**
+ * Intercepts GET request to get object storage endpoints.
+ *
+ * @returns Cypress chainable.
+ */
+export const mockGetObjectStorageEndpoints = (): Cypress.Chainable<null> => {
```

Comment:
```suggestion
export const interceptGetObjectStorageEndpoints = (): Cypress.Chainable<null> => {
```
(Since this util only intercepts the outgoing request and doesn't mock the response)